src/types/notifyApiSecrets.ts

```diff
@@ -1,5 +1,4 @@
-// LEGACY FILE - CODE HAS MOVED TO RESULTS_DELIVERY REPO
-export interface NotifyApiSecrets {
-  notifyApiKey: string
-  notifyTemplateId: string
-}
+// export interface NotifyApiSecrets {
```

Comment:
Just delete this file, think it's still just commented out just now

Reply:
Knew I'd miss one, well done for spotting the deliberate mistake...